app/views/landing-pages/league/PageLeagueGlobal.vue

```diff
@@ -683,7 +683,7 @@ export default {
         <div>
           <img
             class="img-responsive"
-            src="/images/pages/league/sunfire-blitz.png"
+            src="/images/pages/league/system-shock.png"
```

Comment:
```suggestion
            src="/images/pages/league/supercharged.png"
```
We actually don't show this until the championship version is active (see line 653), so this section is only applicable to championship arenas.